test/aave-v2/TestSupplyVault.t.sol

```diff
@@ -161,4 +161,77 @@ contract TestSupplyVault is TestSetupVaults {
         assertEq(token.balanceOf(address(daiSupplyVault)), _deal - _toTransfer);
         assertEq(token.balanceOf(_to), _toTransfer);
     }
+
+    // TODO: fix this test by using updated indexes in previewMint
+    // function testShouldMintCorrectAmountWhenMorphoPoolIndexesOutdated() public {
+    //     uint256 amount = 10_000 ether;
+
+    //     vaultSupplier1.depositVault(daiSupplyVault, amount);
+
+    //     vm.roll(block.number + 100_000);
+    //     vm.warp(block.timestamp + 1_000_000);
+
+    //     uint256 assets = vaultSupplier2.mintVault(daiSupplyVault, amount);
+    //     uint256 shares = vaultSupplier2.withdrawVault(daiSupplyVault, assets);
+
+    //     assertEq(shares, amount, "unexpected redeemed shares");
+    // }
+
+    function testShouldDepositCorrectAmountWhenMorphoPoolIndexesOutdated() public {
+        uint256 amount = 10_000 ether;
+
+        vaultSupplier1.depositVault(daiSupplyVault, amount);
```

Comment:
why this one is depositing assets?

Reply:
Because there need to be assets on the vaults for `totalAssets` to be incorrect